controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -479,6 +508,9 @@ func (r *StorageConsumerReconciler) reconcileCephRadosNamespace(
 				}
 				rns.Spec.Name = radosNamespaceName
 				rns.Spec.BlockPoolName = bp.Name
+				if forceDelete {
+					util.AddAnnotation(rns, util.RookForceDeletionAnnotationKey, strconv.FormatBool(true))
+				}
```

Comment:
Please remove this directly below the owner ref because this is metadata and not spec